src/services/BriefService.ts

```diff
@@ -0,0 +1,12 @@
+import { IBrief } from '../interfaces/Brief';
+import { BriefStatusesConfig } from '../constants/BriefStatuses';
+
+export const getFilteredBriefList = (briefList: IBrief[], filterValue: string): IBrief[] => {
```

Comment:
what if you start having more than 1 filter? You will need to rename `filterValue` to something more specific. We can already name it `statusFilter`

Reply:
or we can keep it like this, but then rename function to `getFilteredBriefListByStatus`